lit_tests/kernel/wave/attention.py

```diff
@@ -1181,3 +1125,59 @@ def test_paged_flash_decoding():
     # CHECK-COUNT-1:           arith.addf
     # CHECK-COUNT-1:      arith.divf
     # CHECK-COUNT-1:      vector.store
+
+
+@run_test
+def test_prefill_attention():
+    shape = AttentionShape(
+        num_query_heads=16,
+        num_kv_heads=4,
+        head_size=64,
+        head_size_kv=64,
+        num_seqs=2,
+        max_seq_len=12,
+        total_seq_len=20,
+    )
+    q_shape = (shape.total_seq_len, shape.num_query_heads, shape.head_size)
+    k_shape = (shape.total_seq_len, shape.num_kv_heads, shape.head_size)
+    v_shape = (shape.total_seq_len, shape.num_kv_heads, shape.head_size_kv)
+    o_shape = (shape.total_seq_len, shape.num_query_heads, shape.head_size_kv)
+    mfma_variant = (tkw.MMAType.F32_16x16x16_F16,) * 2
+    prefill_attention, hyperparams = get_prefill_attention_kernel(
+        shape, mfma_variant, q_shape, k_shape, v_shape, o_shape
+    )
+
+    with tk.gen.TestLaunchContext(
+        hyperparams,
+        canonicalize=True,
+        run=False,
+        run_bench=False,
+        schedule=False,
+        use_scheduling_barriers=False,
+    ):
+        torch.manual_seed(0)
+        q = torch.randn(q_shape, dtype=torch.float16)
+        k = torch.randn(k_shape, dtype=torch.float16)
+        v = torch.randn(v_shape, dtype=torch.float16)
+        output = torch.zeros(o_shape, dtype=torch.float32)
+        offsets = torch.ones(shape.num_seqs, dtype=torch.int32)
+        seq_lens = torch.ones(shape.num_seqs, dtype=torch.int32)
+        print(prefill_attention(q, k, v, offsets, seq_lens, output).module_op)
+
+        # CHECK-LABEL:       func.func @prefill_attention
```

Comment:
Ahhh I think the lit test here needs to be re-worked, IIRC there shouldn't be vector.maskedloads after the reordering patch that we pushed up

Reply:
yeah they have been reordered. I updated the lit test so it should pass now.